### PR TITLE
Docs: Update alert data model of webhook-notifier documentation 

### PR DESCRIPTION
**Pull request by hoptical**
**What is this feature?**
Updates [alert data model of webook-notifer](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/webhook-notifier/#alert) documentation. There are some fields in the [alert data model](https://github.com/grafana/alerting/blob/793c67215ba14d6515b93707caeb7e6d9bf82189/templates/template_data.go#L29) which are not listed in the documentation:
- values
- imageURL
- embeddedImage

Additionally, the `valueString` field has been marked as being deprecated in Grafana 10.  

**Why do we need this feature?**

Webhook developers need to have the full JSON documentation of the Alerting data model.

**Who is this feature for?**

Webhook developers

**Special notes for your reviewer:**

I have considered `values` as an object. Please check if it's correct and if you're Ok with the deprecating note on the `valueString` field.

Please check that:
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 1]
```diff
@@ -110,12 +110,15 @@ Example JSON body:
...
-| valueString  | string | Values that triggered the current status                                           |
+| valueString  | string | **Will be deprecated in Grafana 10**                                               |  
+| values       | object | Values that triggered the current status                                           |
 | generatorURL | string | URL of the alert rule in the Grafana UI                                            |
 | fingerprint  | string | The labels fingerprint, alarms with the same labels will have the same fingerprint |
 | silenceURL   | string | URL to silence the alert rule in the Grafana UI                                    |
 | dashboardURL | string | **Will be deprecated soon**                                                        |
 | panelURL     | string | **Will be deprecated soon**                                                        |
+| imageURL     | string | URL of the screenshots of the panel alert rule assigned to notifications           |
+| embeddedImage| string | embedded screenshots of the panel alert rule assigned to notifications             |
```
Comment (yuri-tceretian) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:121]: embeddedImage are not populated in webhook. it is used only by email integration

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 2]
```diff
@@ -110,12 +110,15 @@ Example JSON body:
 | annotations  | object | Annotations that are part of this alert, map of string keys to string values       |
 | startsAt     | string | Start time of the alert                                                            |
 | endsAt       | string | End time of the alert, default value when not resolved is `0001-01-01T00:00:00Z`   |
-| valueString  | string | Values that triggered the current status                                           |
+| valueString  | string | **Will be deprecated in Grafana 10**                                               |
```
Comment (yuri-tceretian) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:113]: Oh, I found TODO that we missed https://github.com/grafana/alerting/blob/e067f119be06c7ae1e225d8b4bfc2c832a9b23e3/templates/template_data.go#L129 @grobinson-grafana should we delete it now?

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 3]
```diff
@@ -110,12 +110,15 @@ Example JSON body:
...
 | endsAt       | string | End time of the alert, default value when not resolved is `0001-01-01T00:00:00Z`   |
-| valueString  | string | Values that triggered the current status                                           |
+| valueString  | string | **Will be deprecated in Grafana 10**                                               |  
+| values       | object | Values that triggered the current status                                           |
 | generatorURL | string | URL of the alert rule in the Grafana UI                                            |
 | fingerprint  | string | The labels fingerprint, alarms with the same labels will have the same fingerprint |
 | silenceURL   | string | URL to silence the alert rule in the Grafana UI                                    |
 | dashboardURL | string | **Will be deprecated soon**                                                        |
 | panelURL     | string | **Will be deprecated soon**                                                        |
+| imageURL     | string | URL of the screenshots of the panel alert rule assigned to notifications           |
```
Comment (yuri-tceretian) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:120]: I would rephrase it. It is `URL of a screenshot of a panel assigned to the rule that created this notification`

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 1 above)
Comment (hoptical) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md:121]: Ok, I will remove it.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 3 above)
Comment (hoptical) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md:120]: I will rephrase it.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 1 above)
Comment (hoptical) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md]: Done.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 2 above)
Comment (hoptical) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md:119]: Waiting for the response.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 3 above)
Comment (hoptical) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md]: Done.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 4]
```diff
@@ -41,7 +41,10 @@ Example JSON body:
       "silenceURL": "https://play.grafana.org/alerting/silence/new?alertmanager=grafana&matchers=alertname%3DT2%2Cteam%3Dblue%2Czone%3Dus-1",
       "dashboardURL": "",
       "panelURL": "",
-      "valueString": "[ metric='' labels={} value=14151.331895396988 ]"
+      "values": {
```
Comment (hoptical) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:44]: @yuri-tceretian 
I have also added `values` field to the example json body.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md (hunk 2 above)
Comment (yuri-tceretian) [reply docs/sources/alerting/manage-notifications/webhook-notifier.md:119]: Let's delete `valueString` from the doc at all. We will delete it in code after that.

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 5]
```diff
@@ -41,7 +41,9 @@ Example JSON body:
...
+        "data-avg": 14151.331895396988
+      }
```
Comment (yuri-tceretian) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:46]:
```suggestion
        "values": {
        "B": 44.23943737541908,
        "C": 1
      }
```
I think this example will be more intuitive. Because not many users rename their queries\expressions

File: docs/sources/alerting/manage-notifications/webhook-notifier.md [hunk 6]
```diff
@@ -62,7 +64,9 @@ Example JSON body:
       "silenceURL": "https://play.grafana.org/alerting/silence/new?alertmanager=grafana&matchers=alertname%3DT1%2Cteam%3Dblue%2Czone%3Deu-1",
       "dashboardURL": "",
       "panelURL": "",
-      "valueString": "[ metric='' labels={} value=47043.702386305304 ]"
+      "values": {
+        "data-avg": 47043.702386305304
+      }
```
Comment (yuri-tceretian) [inline docs/sources/alerting/manage-notifications/webhook-notifier.md:69]:
```suggestion
      "values": {
        "B": 44.23943737541908,
        "C": 1
      }
```

**Comment by yuri-tceretian** [comment]
Please ` yarn run prettier:write` to make linter happy.

**Review by yuri-tceretian — APPROVED** [review]
LGTM

**Comment by hoptical** [comment]
All suggestions and linting applied.

**Review by brendamuir — APPROVED** [review]
